generators/express/templates/deployment/kubernetes/chart/deployment.yaml.ejs

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "<%= projectName %>.fullname" . }}-<%= packageName %>
+  labels:
+    app.kubernetes.io/component: <%= packageName %>
+    {{- include "<%= projectName %>.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: <%= packageName %>
+      {{- include "<%= projectName %>.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/database-secret: {{ include (print $.Template.BasePath "/<%= packageName %>-database-secret.yaml") . | sha256sum }}
+      labels:
+        app.kubernetes.io/component: <%= packageName %>
+        {{- include "<%= projectName %>.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: node
+          image: {{ include "<%= projectName %>.<%= packageName %>.image" . }}
+          env:
+            - name: DATABASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "<%= projectName %>.fullname" . }}-<%= packageName %>-database
+                  key: url
+            - name: ENVIRONMENT_NAME
+              value: {{ .Values.environment_name }}
+            - name: SENTRY_DSN
+              value: {{ .Values.<%= varify(packageName) %>.sentry.dsn }}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
```

Comment:
It would be great later on to have a real url on backends doing a database connection (or a connection to all right adapters) to ensure readyness to receive trafic

Reply:
That would indeed be great, I'm adding an issue for latter.